consensus/clique/src/test/java/tech/pegasys/pantheon/consensus/clique/blockcreation/CliqueBlockCreatorTest.java

```diff
@@ -197,16 +197,16 @@ public void insertsNoVoteWhenAtEpoch() {
     // ensure that the next block is epoch
     epochManager = new EpochManager(1);
 
-    final CliqueExtraData extraData =
-        new CliqueExtraData(BytesValue.wrap(new byte[32]), null, validatorList);
+    final BytesValue extraData =
+        CliqueExtraData.createUnsignedData(BytesValue.wrap(new byte[32]), validatorList);
```

Comment:
Renamed it to `createWithoutProposerSeal` which is quite literally true.  I suspect there's room for improvement in this overall design to help with the mining process (probably even more so in IBFT) but I'm already changing enough.  I introduced this just because I really hate passing `null` with no explanation and now it needs to be given two `null` values.